EIPS/eip-5627.md

```diff
@@ -0,0 +1,219 @@
+---
```

Comment:
Please update this to use the current EIP format. See the [files tab](https://github.com/ethereum/EIPs/pull/5627/files) for the necessary changes.